yt_dlp/extractor/youtube.py

```diff
@@ -451,6 +451,23 @@ class YoutubeBaseInfoExtractor(InfoExtractor):
 
     _IGNORED_WARNINGS = {'Unavailable videos will be hidden during playback'}
 
+    _YT_HANDLE_RE = r'@[A-Za-z0-9_.-]{3,30}'  # https://support.google.com/youtube/answer/11585688?hl=en
+    _YT_CHANNEL_UCID_RE = r'UC[A-Za-z0-9_-]{22}'
```

Comment:
```suggestion
    _YT_HANDLE_RE = r'@[\w.-]{3,30}'  # https://support.google.com/youtube/answer/11585688?hl=en
    _YT_CHANNEL_UCID_RE = r'UC[\w-]{22}'
```

Reply:
Is YT actually sending strings with Unicode alphanumerics outside `[a-zA-Z\d]`?

Reply:
> Is YT actually sending strings with Unicode alphanumerics outside `[a-zA-Z\d]`?

https://www.youtube.com/channel/UCiu-3thuViMebBjw_5nWYrA

https://wiki.archiveteam.org/index.php/YouTube/Technical_details

Reply:
Sure, I wasn't thinking of the punctuation-like characters. So either `(?A)[\w.-]` (loses in Py2) or just unchanged. The ArchiveTeam link is excellent.